Symfony/CS/Fixer/Contrib/AlignDoubleArrowFixer.php

```diff
@@ -72,15 +73,8 @@ public function getDescription()
      */
     private function injectAlignmentPlaceholders(Tokens $tokens, $startAt = null, $endAt = null)
     {
-        if (empty($startAt)) {
-            $startAt = 0;
-        }
-
-        if (empty($endAt)) {
```

Comment:
And make the interface uglier. But I see your point. And now... I can still use default null values :D